Automattic/jetpack
# My Jetpack: Activate plugins in normal mode

#### Changes proposed in this Pull Request:

* Do not use Silent mode when activating plugins from My Jetpack.

This was done to avoid errors caused by the redirects, but now plugins won't redirect by default and it's safe to activate in normal mode.

This way we trigger the `activated_plugin` hook and make sure plugins are properly activated

#### Other information:


#### Jetpack product discussion


p9dueE-5CL-p2

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Remove Jetpack plugin (only deactivating won't do it). On the monorepo you can do `jetpack docker unlink-plugin jetpack`
* Activate and connect only Protect
* Go to My Jetpack
* Activate Backup and confirm the Backup plugin gets activated without errors
* Do the same for Search and Social
* Install Jetpack `jetpack docker link-plugin jetpack`
* Go to My Jetpack and activate "Extras"
* Confirm Jetpack plugin gets activated without errors

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202830416402617
